src/data/docs/commands.md

```diff
@@ -44,3 +44,80 @@ Here's what these reasons mean:
 #### I want to `undo` a partially completed `add` where some of the definitions failed
 
 Just do [`undo`](#undo)!
+
+## `docs`
+
+Use `docs foo` to print the documentation [associated with](#link) definition `foo` (see [documentation literals](language-reference#documentation-literals) to learn more).
+
+## `find` (and relatives)
+
+The [`find`](#find) command and its relatives ([`find.patch`](#find-patch) and [`find.verbose`](#find-verbose)) can be used to explore the definitions in a Unison codebase.
+
+### `find`
+
+With no arguments, the `find` command will list all definitions in the current namespace (including definitions in descendants of the current namespace).
+
+With one or more arguments (e.g., `find foo bar baz`), the command will list definitions in the current namespace with names similar to the arguments:
```

Comment:
Is this right? It's what `help find` says, but `find foo` seems to look outside the current namespace:

```
.> cd .base.Text
.> find empty
```

```

  1. empty : Text
  2. .base.Bytes.empty : Bytes
  3. .base.List.empty : [a]
  4. .base.Map.empty : Map k v
  5. .base.Set.empty : Set k
  6. .base.Trie.empty : Trie k v
  7. .base.test.internals.v1.Test.Report.empty : Report
  8. .base.test.v1.empty : '{Gen} a
```

Reply:
Looks like the help text is out of date; `find` is one of our oldest and hairiest commands though, so this doesn't surprise me.  It does looks like it's prioritizing results in the current namespace, but then also searching outside.